db/c.cc

```diff
@@ -1145,6 +1145,34 @@ rocksdb_column_family_handle_t** rocksdb_create_column_families(
   return c_handles;
 }
 
+rocksdb_column_family_handle_t** rocksdb_create_column_families_with_options(
```

Comment:
Should we implement a TTL variant, or is that enough scope ? I prefer to restrict the scope but worth thinking about.

Reply:
This is the first C API. I would like to start small and then we can add more later. Once API is out, it's hard to remove it.

Reply:
Agree.